docs/design/bundle-sources.md

```diff
@@ -0,0 +1,300 @@
+# Building a file-based catalog from a plain bundle image
+
+> **Warning:** Operator Lifecycle Manager (OLM) v1 features and components are still experimental. Early adopters and contributors should expect breaking changes. The following procedures are not recommended for use on production clusters.
+
+You can build a static collection of arbitrary Kubernetes manifests in the YAML format, or *plain bundle*, and add the image to a file-based catalog (FBC). The experimental `olm.bundle.mediatype` property of the `olm.bundle` schema object differentiates a plain bundle from a regular (`registry+v1`) bundle. You must set the bundle media type property to `plain+v0` to specify a plain bundle.
+
+For more information, see the [Plain Bundle Specification](https://github.com/operator-framework/rukpak/blob/main/docs/bundles/plain.md) in the RukPak repository.
+
+To build a file-based catalog from a plain bundle image, you must complete the following steps:
+
+* Create a plain bundle image
+* Create a file-based catalog
+* Add the plain bundle image to your file-based catalog
+* Build your catalog as an image
+* Publish your catalog image
+
+## Building a plain bundle image from an image source
+
+Currently, the Operator Controller only supports installing plain bundles created from a plain bundle image.
+
+<h3 id="prereqs-building-plain-bundle-from-image">
+Prerequisites
+</h3>
+
+* [`opm` CLI tool](https://github.com/operator-framework/operator-registry/releases)
+* Docker or Podman
+* Push access to an image registry, such as [Quay](https://quay.io)
+
+<h3 id="proc-building-plain-bundle-from-image">
+Procedure
+</h3>
+
+1. Verify that your Kubernetes manifests are in a flat directory at the root of your project similar to the following example:
+
+    ```sh
+    tree manifests
+    manifests
+    ├── namespace.yaml
+    ├── service_account.yaml
+    ├── cluster_role.yaml
+    ├── cluster_role_binding.yaml
+    └── deployment.yaml
+    ```
+
+    * If you are using [kustomize](https://kustomize.io) to build your manifests from templates, you must redirect the output into a single file under the `manifests/` directory by running the following command:
+
+        ```sh
+        kustomize build templates > manifests/manifests.yaml
+        ```
+
+   For more information, see [Building a plain bundle > Prerequisites](https://github.com/operator-framework/rukpak/blob/main/docs/bundles/plain.md#prerequisites).
+
+1. Create a Dockerfile at the root of your project:
+
+    ```sh
+    touch Dockerfile.plainbundle
+    ```
+
+1. Make the following changes to your Dockerfile:
+
+    *Example Dockerfile*
+
+    ```sh
+    FROM scratch
+    COPY manifests /manifests
+    ```
+
+    > **Note:** Use the `FROM scratch` directive to make the size of the image smaller.
+
+1. Add the following labels to your Dockerfile:
+
+    ```sh
```

Comment:
Does anything in OLMv1 read these image labels? If not, I'd suggest dropping the ones that are not explicitly used in OLMv1 code.

Reply:
Fair point, my understanding was that these labels were required for the `opm validate` command which may not be necessary considering your comment about removing it from the steps due to the OLMv0 assumptions it makes.

Reply:
Good point, these image labels aren't being used anywhere in OLMv1 so dropped all of them. I've added them originally assuming these are required for `opm render` but realized we don't have support for plain bundles via `opm` yet.